pymongo/errors.py

```diff
@@ -146,9 +146,16 @@ def __init__(self, error, code=None, details=None):
             error_labels = details.get('errorLabels')
         super(OperationFailure, self).__init__(
             error, error_labels=error_labels)
+        self.__error = error
         self.__code = code
         self.__details = details
 
+    @property
+    def error(self):
+        """The error string returned by the server, if any.
+        """
+        return self.__error
```

Comment:
Same question about `self._message`.

Reply:
Ah, yeah that would probably be a better method. Let me try that.